substrate/frame/bounties/src/lib.rs

```diff
@@ -221,9 +224,12 @@ pub mod pallet {
 		#[pallet::constant]
 		type BountyDepositPayoutDelay: Get<BlockNumberFor<Self, I>>;
 
-		/// Bounty duration in blocks.
+		/// Optional bounty duration in blocks.
+		///
+		/// If `None`, bounties stay active indefinitely, removing the need for
+		/// `extend_bounty_expiry` and restricting curator slashing to OpenGov.
```

Comment:
```suggestion
		/// If `None`, bounties stay active indefinitely, removing the need for
		/// `extend_bounty_expiry`.
```

OpenGov part is specific to the runtime e